src/agents/run.py

```diff
@@ -59,8 +59,9 @@
 
 DEFAULT_MAX_TURNS = 10
 
-DEFAULT_AGENT_RUNNER: AgentRunner = None  # type: ignore
+DEFAULT_AGENT_RUNNER: AgentRunner | None = None
```

Comment:
this is correct for the line but as mentioned in the following line, the value is set during the loading time. so no need to make this change